src/cell/utility.c

```diff
@@ -83,6 +85,47 @@ jsal_require_pathname(int index, const char* origin_name)
 	return path_cstr(path);
 }
 
+bool
+dircopy(const char* src, const char* dest)
+{
+	tinydir_dir dir;
+	if(tinydir_open(&dir, src) == -1) return false;
```

Comment:
Nit: Space after `if`, also move `return` to the following line and indent.

Reply:
I was usually careful about copying your style for consistency but I guess I missed that space and newline. There's also a missing newline in wildcmp() on line 206.